auth_server/authn/tokendb.go

```diff
@@ -22,11 +22,11 @@ import (
 	"fmt"
 	"time"
 
-	"golang.org/x/crypto/bcrypt"
-
+	"github.com/cesanta/docker_auth/auth_server/api"
```

Comment:
please order imports in 3 groups: system, 3rdparty, then local. this should be at the bottom of the list.

Reply:
Fixed